custom/ilsgateway/tests/handlers/utils.py

```diff
@@ -112,6 +114,7 @@ def tearDownClass(cls):
         SQLLocation.objects.all().delete()
         generator.delete_all_subscriptions()
         Domain.get_by_name(TEST_DOMAIN).delete()
+        super(TestScript, super(ILSTestScript, cls)).tearDownClass()
```

Comment:
woa, that's quite weird. Never seen that pattern before. What's the issue with calling the `ILSTestScript` superclass methods?